packages/react-native/migrations.json

```diff
@@ -627,6 +627,47 @@
           "alwaysAddToPackageJson": false
         }
       }
+    },
+    "19.0.0": {
+      "version": "19.0.0-beta.0",
```

Comment:
Can you please update these migration versions to `19.0.0-beta.6` since `19.0.0-beta.5` is already out please?